requirePromise.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 module.exports = function requirePromise() {
-	if (typeof Promise !== 'function') {
+	if (typeof (Promise || global.Promise || window.Promise) !== 'function') {
```

Comment:
I tried many ways to isolate the problem, but I couldn't make it.
I am using Meteor with a couple of babel plugins. I have no idea what is breaking the context inside the module.